### DISCUSSION TITLE
Option no logging PTR?

### DISCUSSION THREAD

**Discussion by TheCeet** [AdguardTeam/AdGuardHome#2985] [Q&A]
Is it possible to stop logging PTR requests?
Every 10 minutes I've got a mini tsunami `*reverseIP*in-addr.arpa` requests.

It will be handy that PTR requests are happening, but that they are not in the query log.

**Comment by ameshkov** [comment]
Hm, are they new all the time? It should've been caching results and not making requests for the same IP twice

**Reply by TheCeet** [reply]
Every 10min I get PTR requests from my devices.
See: https://imgur.com/a/V3VvCnb

I've 2 AdGuard Homes on 2 Raspberry Pi's.

**Reply by ameshkov** [reply]
@EugeneOne1 could you please take a look?

**Reply by TheCeet** [reply]
More screenshots:
This device it currently not anymore on the network. But it repeats every 10mins:
https://imgur.com/a/S4yzhpd

This device is 24/7 connected:
https://imgur.com/a/JueMhI0

I don't know if this is normal behaviour or not.
If yes, maybe it would be usefull if there's an option stop logging PTR requests in the querylog (if possible)

AdGuardHome version 0.105.2 on 
- Raspberry PI OS on Pi4 (UTP)
- DietPI OS on Raspberry pi Zero W (wifi)

DNS upstream servers:
127.0.0.1:5335 (unbound)
[/168.192.in-addr.arpa/]192.168.50.1

**Reply by ameshkov** [reply]
Hmm, I guess we could make the re-check period considerably lower.

**Comment by EugeneOne1** [comment]
We've investigated the issue.  These PTRs are coming from one of your _AdGuard Home_ instances to the other.  This is the expected behavior.

There is a [feature request](https://github.com/AdguardTeam/AdGuardHome/issues/2939) about searching the query log for specific requests' types.  It may become an alternative solution for you since optional disabling of PTR logging doesn't really seem to be practical enough.

**Reply by dredjohn21** [reply]
@EugeneOne1 & @ameshkov 

I have +1 this feature request to allow suppression.  I think this will be helpful for some users who don’t want this activity clogging their SD card.

Unfortunately, my Linksys Velop is a mesh setup and is very chatty.  I had over 40k of these PTR requests in 12 hours on one node and 20k on another.  From my Reddit and forum searches I see other Velop clients looking for similar suppression solutions.  The consensus is the activity is “normal”, and if I’m just going to ignore, I would prefer to not create noise on the dashboard and also not clog my sd card since I would like to store weeks of data if possible.

**Reply by ainar-g** [reply]
@dredjohn21, if you want to block processing of those completely, you can add the subnet domains to the domain blocklist on the _DNS settings_ page.  That makes AGH drop any queries for those domains immediately, before any processing takes place.

**Reply by dredjohn21** [reply]
> @dredjohn21, if you want to block processing of those completely, you can add the subnet domains to the domain blocklist on the _DNS settings_ page. That makes AGH drop any queries for those domains immediately, before any processing takes place.

Thanks for guidance.  Newbie questions:
1. Does blocking prevent the activity?  Or just stops logging?  (I don’t want to break whatever my router is doing… just don’t want to see it/record it, if the group think is it is normal and benign)
2. “If” the blocking works as desired, do you have any coaching on how to block just the “in-addr.arpa” activity vs everything on the device?  I’m assuming this is a regex activity, and haven’t done that before… 

Appreciate the help!

**Reply by dredjohn21** [reply]
@ainar-g Brief update: I think I figured it out.  I went to DNS Settings->Disallowed Domains and added a 4th entry:

version.bind
id.server
hostname.bind
***.in-addr.arpa**

This has eliminated the logging (great news - thank you).  And I'm also not noticing any issues with my network (great news).  However, I'm not 100% clear on how these requests are being handled now.

The instruction state "AdGuard Home drops DNS queries matching these domains, and these queries don't even appear in the query log."  Does this mean my router is no longer able to do these domain lookups?  Or, it's performing the action, but AdGuard is just ignoring the activity?  

Basically, just looking for professional reassurance I haven't messed up my system :)

**Reply by ainar-g** [reply]
Yes, that means that PTR queries are blocked completely.  If you want to block only some of the queries, you can add the subnet ARPA domains there.  For example, to block only the `192.168.X.Y` lookups, add `||168.192.in-addr.arpa`.  And so on.

**Reply by dredjohn21** [reply]
> Yes, that means that PTR queries are blocked completely. If you want to block only some of the queries, you can add the subnet ARPA domains there. For example, to block only the `192.168.X.Y` lookups, add `||168.192.in-addr.arpa`. And so on.

Thanks so much @ainar-g 

I made that change and am now seeing some other traffic come through (I checked one and it was an Amazon AWS lookup)

I’m assuming this is better to ensure the right monitoring of external traffic

(and hope no issues will pop up with the disallow for internal lookups - still not 100% clear what is happening with those 192.168* lookups, but won’t dwell if I’m not noticing any issues)

Again appreciate your advice/help and hope if others stumble on this in future the guidance will be helpful
